build_deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+IMAGE_BUILD="ekidd/rust-musl-builder:1.30.1"
+
+IMAGE="quay.io/app-sre/cincinnati"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+DOCKERFILE_DEPLOY="dist/Dockerfile"
+
+# clean up
+rm -rf target
+
+docker run --rm -v $(pwd):/home/rust/src $IMAGE_BUILD cargo build --release
```

Comment:
is there a way not to hardcode this path, or do you assume this is never run locally?

Reply:
sorry, which path?
`/home/rust/src` this is just a path inside  `ekidd/rust-musl-builder:1.30.1` image it should work locally same way as on CI, moreover you can build your project locally without this script since I assume you have rust installed, this neeed more for CI where we want to run some build in docker to avoid of installing alot of deps on CI slaves.

Reply:
Ah right. I was looking too quickly and didn't catch that. Sorry for the fuss